docs/index.rst

```diff
@@ -20,7 +20,18 @@ API Reference
     :maxdepth: 2
 
     datacatalog_v1beta1/services
-    datacatalog_v1beta1/types
+   datacatalog_v1beta1/types
```

Comment:
Owlbot keeps removing this. This commit tried to add it back but possibly I need to add it back somewhere else so it doesn't get overwritten? @tswast WWYD?